srml/bridge/src/lib.rs

```diff
@@ -303,4 +337,90 @@ mod tests {
 			assert_eq!(MockBridge::num_bridges(), 1);
 		});
 	}
+
+	fn get_related_block_headers() -> (Header, Header, Header) {
```

Comment:
A helper function like `fn build_header_chain(first_parent_hash: HashOutput, len: usize) -> Vec<Header>` would probably make tests more flexible and concise.